ext/sockets/conversions.c

```diff
@@ -241,26 +241,25 @@ static void from_zval_write_aggregation(const zval *container,
 	zval					*elem;
 
 	if (Z_TYPE_P(container) != IS_ARRAY) {
-		do_from_zval_err(ctx, "%s", "expected an array here");
+		zend_type_error("Expected array");
```

Comment:
I don't really have a clue, at one point I was trying to refactor it to use a HashTable param instead of a Zval so that I could track from where it was coming, but ran into issues as the function pointers in the struct expect a Zval IIRC.

I imagine refactoring this part of the code may be possible but I clearly don't have enough understanding of C to perform it nor do I know about networking that much.

So maybe we could drop this change and try to change it in an isolated fashion with a refactor?

Reply:
Yes, dropping this change would be a good idea